docs/Concepts/Privacy/Privacy-Groups.md

```diff
@@ -1,76 +1,86 @@
-description: Privacy
+description: Privacy groups
 <!--- END of page meta data -->
 
-# Privacy Groups 
+# Privacy groups
 
-A privacy group is a group of nodes identified by a unique privacy group ID by Orion. Each private transaction is 
-stored in Orion with the privacy group ID.
+A privacy group is a group of nodes identified by a unique privacy group ID by Orion. Each private
+transaction is stored in Orion with the privacy group ID.
 
-The Besu nodes maintain the public world state for the blockchain and a private state for each privacy group. 
-The private states contain data that is not shared in the globally replicated world state. 
+The Besu nodes hold the public world state for the blockchain and a private state for each privacy
```

Comment:
I think this needs to be _maintain_ rather than _hold_ -> the public and private state are updated in the Besu node rather than being a static state which I think _hold_ implies

Reply:
Fixed